etna/collections/models.py

```diff
@@ -169,7 +173,14 @@ class Meta:
         verbose_name_public = _("explore by topic")
 
     featured_article = models.ForeignKey(
-        "articles.ArticlePage", blank=True, null=True, on_delete=models.SET_NULL
```

Comment:
I know we didn't originally have `verbose_name=_("featured articles")` here, but it'd be good to add it 😃